sickbeard/name_parser/regexes.py

```diff
@@ -116,7 +116,7 @@
     ('stupid',
      # tpz-abc102
      r'''
-     (?P<release_group>.+?)-\w+?[\. ]?           # tpz-abc
+     (?P<release_group>.+?)(?<!WEB)-\w+?[\. ]?           # tpz-abc
```

Comment:
Very very unlikely, topaz the main group using this file naming, sort of like FoV had a diferent one quit releases like 6 y ago before web-dls where a thing.
And if it is a web-dl by chance the actual file name wont have it 
Ex file name `grp-shw101` wil be the equivelant to show.S01e01.some quality-group , they use shortend versions of the show's title and group name.
Hopefully i made myself understandable

`tpz-abc102` is the file name equivelant to the dir name `Abc.S01E02.somequality-topaz